tables/translated/kRO_english/STATUS_id_handle.txt

```diff
@@ -0,0 +1,1201 @@
+# see kRO client: luafiles514\lua files\stateicon\EFSTIDs.lub
```

Comment:
I think this is an extra file. We can use a common file: `tables\STATUS_id_handle.txt`